include/pmacc/eventSystem/streams/EventStream.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2013-2022 Heiko Burau, Rene Widera, Alexander Grund
+/* Copyright 2013-2022 Felix Schmitt, Rene Widera, Benjamin Worpitz
```

Comment:
What happened here with the authors?

Reply:
The files got removed but github tries to link files and is thinking we renamed the file.
So this change is OK.